### ISSUE TITLE
Flow complains

### ISSUE THREAD

**Issue by josephbuchma** [zoontek/react-native-permissions#230]
### Describe your environment

* React-Native version *0.51.0*
* react-native-permissions version *1.1.1*
* Flow version *5.5.1*

### How to repeat issue and example

`npm install --save react-native-permissions` and use it. Flow complains:

```
Error: node_modules/react-native-permissions/index.js:3
  3: import Permissions from './lib/permissions'
                             ^^^^^^^^^^^^^^^^^^^ ./lib/permissions. Required module not found
```

**Comment by zoontek** [comment]
`Flow version 5.5.1`? How is this possible?

**Comment by dikarel** [comment]
I'm having the same issue on Flow `0.6.1`; looks like Flow is not happy about this library using `index.ios.js` and `index.android.js`

**Comment by josephbuchma** [comment]
@dikarel yeah, with `index.js` it works.

**Comment by zoontek** [comment]
It's now available in the `1.2.0`! Please check the changelog https://github.com/react-native-community/react-native-permissions/releases/tag/1.2.0
